packages/botonic-react/src/webchat/webchat.jsx

```diff
@@ -636,10 +638,16 @@ export const Webchat = forwardRef((props, ref) => {
   )
   const webchatReplies = () => <WebchatReplies replies={webchatState.replies} />
 
-  const userInputEnabled = getThemeProperty(
-    'userInput.enable',
-    props.enableUserInput !== undefined ? props.enableUserInput : true
-  )
+  const isUserInputEnabled = () => {
+    const isUserInputEnabled = getThemeProperty(
+      'userInput.enable',
+      props.enableUserInput !== undefined ? props.enableUserInput : true
+    )
+    if (!isUserInputEnabled) return false
+    return !webchatState.isCoverComponentOpen
```

Comment:
Could be simplified to:
```suggestion
    return isUserInputEnabled && !webchatState.isCoverComponentOpen
```

Reply:
@asastre @marcrabat depending on the case, the "early return" (see section at our [best practices](https://docs.google.com/document/d/1-qd2FdXbAe7XEbdTGyACeCs9HRCauCogocfEU-h-n-I/edit#heading=h.e072veoorbnb)) may be better.

otoh, Adria's version is shorter which is better. 

so it's a draw, which means that both versions are ok imho